drivers/include/net/netdev.h

```diff
@@ -310,19 +310,26 @@ typedef struct netdev_driver {
     int (*send)(netdev_t *dev, const iolist_t *iolist);
 
     /**
-     * @brief Get a received frame
+     * @brief Drop a received frame, *OR* get the length of a received frame,
```

Comment:
Maybe even make the **OR** bold?
```suggestion
      * @brief Drop a received frame, **OR** get the length of a received frame,
```